terraform/files/deploy.sh

```
@@ -0,0 +1,139 @@
+set -e
+
+# Configuration
+APP_DIR="/opt/canadaspends"
+REPO_URL="https://github.com/BuildCanada/CanadaSpends.git"
+BRANCH=${1:-main}  # Use first argument as branch, default to main
+
+echo "Deploying $BRANCH branch to server"
+
+# Load environment variables from .env file if it exists
+if [ -f "$APP_DIR/.env" ]; then
+  export $(grep -v '^#' $APP_DIR/.env | xargs)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)